andela/ah-frontend-thanos
# #161348769 Users can search and/or filter Articles

#### What does this PR do?
Enables users to search and/or filter by `title`, `tag`, and `author`
#### Description of Task to be completed?
- Search by title, author, tag
- Filter by title, author, tag
- Search/Filter by any two, or all three above
#### How should this be manually tested?
- Type your search query in the search field and the results will show up on the page.
- To filter, use any of the filter fields (title, tag, or author) that will automatically appear on the page
#### What are the relevant pivotal tracker stories?
[#161348769](https://www.pivotaltracker.com/story/show/161348769)
#### Screenshots
![image](https://user-images.githubusercontent.com/36793346/50237355-bf0d2700-03cd-11e9-9895-beb6d398702b.png)
